cpp/models/ide_secir/model.h

```diff
@@ -195,6 +196,16 @@ class Model
      */
     void compute_recovered();
 
+    /**
+     * @brief Setter for total number of confirmed cases at time t0.
+     * 
+     * @param[in] total_confirmed_cases Total confirmed cases at time t0 can be set if it should be used for initialisation.
+     */
+    void set_total_confirmed_cases(ScalarType total_confirmed_cases)
+    {
+        m_total_confirmed_cases = total_confirmed_cases;
+    }
+
```

Comment:
What is the use case for this function? I would think that we either know total_confirmed_cases at the beginning of the simulation or not. If we know it, we can include it directly in the construction of the model and don't need to set this value at a later point but I may be missing something.

Reply:
Yes, this setter is not used, i deleted it